lib/Client.php

```diff
@@ -271,6 +284,46 @@ private function send($key, $value, $type, $sampleRate)
     }
 
     /**
+     * @param $tags
+     * @return string
+     */
+    private function generateTagsMessage($tags)
+    {
+      // Nothing to return if no tags
+      if (empty($tags)) {
+        return '';
+      }
+
+      // Normalize string inputs into an array
+      if (!is_array($tags)) {
+        $tags = [$tags];
```

Comment:
do you really want to do this?  you'll end up with a tag whose name is "0" and whose value is the bogus thing passed in.